torch/_dynamo/variables/builder.py

```diff
@@ -582,9 +591,23 @@ def build_key_value(i, k, v):
 
                 return key, value
 
-            result = dict(
-                build_key_value(i, k, v) for i, (k, v) in enumerate(value.items())
-            )
+            def build_set_keys(i, k):
+                if all_const:
+                    key = ConstantVariable.create(k)
+                    source_key = k
+                else:
+                    source_key = ConstDictKeySource(self.get_source(), i)
+                    key = LazyVariableTracker.create(k, source_key)
+
+                source_value = GetItemSource(self.get_source(), source_key)
```

Comment:
This is wrong. There is no such thing as `set[index]`.

Reply:
ah yeh no value is needed this line should be removed.